.gitignore

```diff
@@ -17,3 +17,8 @@
 .cxx
 local.properties
 .envrc
+
+# smoke test artifacts
+smoke-tests/collector/data-results/data-ios-test.json
+smoke-tests/collector/data.json
+smoke-tests/report.xml
```

Comment:
Would `smoke-tests/collector/data-results/*` make sense here? 
I don't think  we'd ever want to check in `data-*-test.json`

Reply:
wow thanks for catching my copy paste mistake